packages/dmn-js-shared/test/spec/components/EditableComponentSpec.js

```diff
@@ -91,6 +91,19 @@ describe('components/EditableComponent', function() {
   });
 
 
+  it('should render with label', function() {
+
+    // when
+    const node = renderToNode(
+      <TestComponent value={ null } label="label" />
+    );
+
+    // then
+    expect(node.innerHTML).to.match(/aria-label="label"/);
+  });
+
+
+
   describe('empty class', function() {
```

Comment:
Code Style
```suggestion
  });


  describe('empty class', function() {
```

Reply:
If only there was a linting rule for this 🤡

Reply:
It is [you to create one](https://github.com/bpmn-io/eslint-plugin-bpmn-io) :wink:.